src/python/pants/option/options_bootstrapper.py

```diff
@@ -100,7 +101,12 @@ def create(cls, env=None, args=None):
     short_flags = set()
 
     def filecontent_for(path):
-      return FileContent(ensure_text(path), read_file(path, binary_mode=True))
```

Comment:
This was probably an abuse of FileContent... won't ask to remove it, but would you mind adding a TODO saying that this codepath doesn't really need to be using FileContent, per-se?

Reply:
Done